### PR TITLE
Thruk2 theme correction to use status_color_background

### PR DESCRIPTION
**Pull request by alexstaz**
Just a little update to allow the use of the option status_color_background in the Thruk2 theme.

### CODE REVIEW

File: themes/themes-available/Thruk2/stylesheets/Thruk2.css [hunk 1]
```diff
@@ -781,14 +781,22 @@ TD.stateInfoTable1 > TABLE,TD.stateInfoTable2 > TABLE {
...
   height:22px;
 }
 
+TD.statusBGCRITICAL a,TD.statusBGCRITICALACK a,TD.statusBGCRITICALSCHED a,TD.statusBGDOWN a,TD.statusBGDOWNACK a,TD.statusBGDOWNSCHED a,TD.statusBGUNREACHABLE a,TD.statusBGUNREACHABLEACK a,TD.statusBGUNREACHABLESCHED a{
+  color: #ffffff;
+}
+
 TD.statusBGUNKNOWN {
+  background-color:#ffcd00
   height:22px;
```
Comment (sni) [inline themes/themes-available/Thruk2/stylesheets/Thruk2.css]: there is a semicolon missing

File: themes/themes-available/Thruk2/stylesheets/Thruk2.css [hunk 2]
```diff
@@ -781,14 +781,22 @@ TD.stateInfoTable1 > TABLE,TD.stateInfoTable2 > TABLE {
...
 }
 
 TD.statusBGWARNING {
+  background-color:#ffe57b
```
Comment (sni) [inline themes/themes-available/Thruk2/stylesheets/Thruk2.css]: here too

**Comment by alexstaz** [comment]
Sorry for the missing semicolon, I have updated the request.

**Comment by sni** [comment]
Looks good, thanks and merged.

**Pull request merged**